Update documentation-government-networking.md

mark-beaupre opened the pull request:
The documentation around the boundary was not specific to what the boundary is. The proposed update specifies the boundary to be an ITAR boundary.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Update documentation-government-networking.md

The documentation around the boundary was not specific to what the boundary is. The proposed update specifies the boundary to be an ITAR boundary.
